dumbfile.json

```diff
@@ -0,0 +1,9 @@
+{
+    "testRegex": "(/tests/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$",
+    "moduleFileExtensions": ["ts", "tsx", "js", "jsx", "json", "node"],
+    "coverageDirectory": "./coverage/",
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+        "**/**/*.{js,jsx}"
+      ]
+}
```

Comment:
I don't seem to understand why we need this dumbfile.json, is it part of the jest setup?

Reply:
I was trying to test out different jest configurations using different test cases. it worked but the file is no longer needed. it will be removed.